tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -126,9 +126,20 @@ kubetest2 kops "${KUBETEST2_ARGS[@]}" \
   --create-args="${create_args[*]}" \
   --control-plane-size="${KOPS_CONTROL_PLANE_SIZE:-1}"
 
+KUBECONFIG=$(mktemp -t kubeconfig)
+export KUBECONFIG
+kops export kubecfg --admin --kubeconfig="${KUBECONFIG}"
+
 if [[ "${RUN_CL2_TEST:-}" == "true" ]]; then
-  # TODO
-  echo "Run CL2 tests"
+  kubetest2 kops "${KUBETEST2_ARGS[@]}" \
+  --test=clusterloader2 \
+  --kubernetes-version="${K8S_VERSION}" \
+  -- \
+  --alsologtostderr \
```

Comment:
Do we need this? `alsologtostderr`

Reply:
Thanks yeah not required since we are not logging to a file, I have removed it.